tests/robustness/coverage/key_pattern_test.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2025 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coverage_test
+
+import (
+	"strings"
+	"testing"
+)
+
+func pattern(k string) (string, bool) {
```

Comment:
```suggestion
func pattern(key string) (string, bool) {
```
What's with the single name letters?

Reply:
Left-over from refactor. Previously `k` was modified version of `key` so it avoided confusion. Updated